### PR TITLE
Added Storage Classes topic for Online for BZ#1411222

### PR DESCRIPTION
**Pull request by ahardin-rh**
Addresses https://bugzilla.redhat.com/show_bug.cgi?id=1411222

This topic is for Online only to prevent content duplication. Other Storage Classes content lives in the Installation and Configuration section of the docs, but Installation and Configuration does not appear in Online documentation.

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@abhgupta @damemi I stubbed this page out. PTAL and let me know if we should add to this. Thanks!

**Comment by ahardin-rh** [comment]
cc @jwforres

**Comment by ahardin-rh** [comment]
also cc @sspeiche

**Comment by abhgupta** [comment]
looks good to me

**Comment by damemi** [comment]
Do we know what the url for this page will be?

**Comment by ahardin-rh** [comment]
@damemi https://docs.openshift.com/online/dev_guide/storage_classes.html

File: dev_guide/storage_classes.adoc [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+requested, as well as provides a means for passing parameters for dynamically
+provisioned storage on demand. `StorageClass` objects can also serve as a
+management mechanism for controlling different levels of storage and access to
+the storage. Cluster administrators (users with `cluster-admin` privileges) or
+storage administrators (users with `storage-admin` privileges) define and create
+the `StorageClass` objects that users can request without needing any intimate
+knowledge about the underlying storage volume sources.
+
+In {product-title}, the storage class is configured and a single option is
+available to the user based on the underlying cloud provider.
```
Comment (ahardin-rh) [inline dev_guide/storage_classes.adoc]: @abhgupta @damemi should we have cloud/storage providers actually listed by name there? Thanks!

**Comment by ahardin-rh** [comment]
@abhgupta @damemi I was discussing this more with my team. Did we want to keep Storage Classes its own page, or do you think this could become a section on this page? https://docs.openshift.com/online/dev_guide/persistent_volumes.html

**Comment by abhgupta** [comment]
Being a subsection of PVs was my recommendation on IRC
dev_guide --> persistent_volumes --> storage_classes

However, this will require some changes to the doc link that we specify and we've just missed the boat on that. Lets coordinate this change for an upcoming devpreview upgrade/release.

**Comment by ahardin-rh** [comment]
[rev_history]
|xref:../dev_guide/storage_classes.adoc#dev-guide-storage-classes[Storage Classes]
|New topic about what Storage Classes are.
%

**Pull request merged**